payload/context/status-set.go

```diff
@@ -64,16 +59,20 @@ func (c *StatusSetCmd) Run(ctx *cmd.Context) error {
 	if err := c.validate(ctx); err != nil {
 		return errors.Trace(err)
 	}
+	hctx, err := c.hookContextFunc()
+	if err != nil {
+		return errors.Trace(err)
+	}
 
-	if err := c.hctx.SetStatus(c.class, c.id, c.status); err != nil {
+	if err := hctx.SetStatus(c.class, c.id, c.status); err != nil {
 		return errors.Trace(err)
 	}
 
 	// TODO(ericsnow) Is the flush really necessary?
 
-	// We flush to state immedeiately so that status reflects the
+	// We flush to state immediately so that status reflects the
```

Comment:
No :) I am sure the same word is mistyped 3 times in the same way in the similar-ish looking code :D